google/cloud/bigtable/internal/completion_queue_impl.h

```diff
@@ -172,7 +172,7 @@ template <typename Request, typename Response, typename Functor,
 class AsyncUnaryRpcFunctor : public AsyncGrpcOperation {
  public:
   explicit AsyncUnaryRpcFunctor(Functor&& functor)
-      : functor_(std::forward<Functor>(functor)) {}
+      : sync_(false), functor_(std::forward<Functor>(functor)) {}
```

Comment:
I'm not sure that the usage of atomic<bool> makes sense, but I don't fully grok the situation either.  Happy to chat about it later.

In the meantime, initialization looks good.

Reply:
Yup, I think a plain mutex would be more readable here anyway.